demo/sdl2surface_rawfb/main.c

```diff
@@ -151,25 +155,35 @@ int main(int argc, char **argv)
 
     printf("desktop display mode %d %d\n", dm.w, dm.h);
 
-
+    /* Of course we need to make a window as well, before anything else */
```

Comment:
Good code comments and technical documentation avoids terms like "Of course", as it is not always obvious to all developers.

Reply:
``` suggestion
    /* Create the window */
```